image/templates/helm/stackrox-central/templates/01-central-15-exposure.yaml

```diff
@@ -41,6 +41,35 @@ spec:
     kind: Service
     name: central
 ---
+
+{{- if and ._rox.central.exposure.route.reencrypt.enabled ._rox._ca }}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: central-reencrypt
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "route" "central-reencrypt") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "route" "central-reencrypt") | nindent 4 }}
+    "helm.sh/hook": "pre-install,pre-upgrade"
```

Comment:
Why do we need the hooks?

Reply:
I copied these from other resources that use `_rox._ca`. But you're right, I think these are not actually needed here.

Reply:
removed